src/helpers.coffee

```diff
@@ -172,7 +172,7 @@ syntaxErrorToString = ->
   marker   = codeLine[...start].replace(/[^\s]/g, ' ') + repeat('^', end - start)
 
   # Check to see if we're running on a color-enabled TTY.
-  if process?
+  if process?.stdout? and process?.env?
```

Comment:
`if process?
  colorsEnabled = process.stdout?.isTTY and not process.env?.NODE_DISABLE_COLORS`?